deploy/Chart/templates/dashboard/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.dashboard.enabled }}
```

Comment:
Is there a better way to do this? (ignore the whole subfolder)

Reply:
We can use subchart, but it is hard to version it. So I would use this way for now.